ferrilata-devilline/Services/Extensions/AdminJsonAnalizer.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ferrilata_devilline.Services.Extensions
+{
+    public static class AdminJsonAnalizer
+    {
+        private class ExpectedFormat
+        {
+            [JsonProperty(Required = Required.Always)]
+            string version { set; get; }
```

Comment:
It would be more consistent if there were explicit access modifiers.

Reply:
...of the variables or the classes? I'm a bit confused, sorry. The inner class is private to show that it is not intended to be used anywhere else.